### DISCUSSION TITLE
在 Solana 上构建一个 Bounties/Hackathon 去中心化发布/参与平台

### DISCUSSION THREAD

**Discussion by bqx619** [creatorsand/.github#10] [Ideas]
## 项目名
**在 Solana 上构建一个 Bounties/Hackathon 去中心化发布/参与平台**

## 项目介绍
1. 目前继 gitcoin 将 bounties 功能转移到 buildbox 之后, 尚无一个代表性的且专注的 bounties 平台, 而 buildbox 本身专注点是 hackathon, 且越来越中心化
2. web3/blockchain 领域有很大一部分 bounty hunter, 且大部分项目方都对一个可自然聚集且人员技能明确的平台有需求
3. 可以很好的串联类似 身份验证/资金支付 等平台
4. 后续可以往 Hackathon/Grants 发展
5. 应当重复提及的是, 前期核心可以是: 
   1. 为需要解决 bounties 的组织/个人提供发布平台和专业技能人才索引/查找平台
   2. 为具备专业技能人才的人寻找感兴趣的项目/赚钱赏金提供渠道
   3. **考虑设计一种能够双向验证资质的设计, 既能一定程度上确定"人"具备解决问题的能力, 又能确定发布 bounties 的"组织"具备可信度**
   4. **考虑提高"人"参与的兴趣和门槛, 比如具备解决问题能力的人, 可以获得独属的 NFT, NFT 具备某些权利**
6. 这不是非 solana 链不可的项目思路, 但从去中心化成本等方面来看, 在 solana 链上, 它可以更具有生命力
7. 愿景是 web3/blockchain 的一种基础设施

## 发起人介绍(社群名)
Shawn(wx: Watcher)

## 项目地址
-/-

### 参考资料
- https://app.buidlbox.io/
- https://dorahacks.io/
- https://codemart.com/projects

**Comment by DaviRain-Su** [comment]
https://build.superteam.fun/ideas/open-grants-protocol-for-solana

**Reply by bqx619** [reply]
这个感觉是目前 gitcoin 的思路, 他们为了 grants 放弃了 bounties 和 hackathon, 为此做了一整套的前端加合约

- 合约: https://github.com/allo-protocol/allo-contracts
- 前端: https://github.com/gitcoinco/grants-stack

**Comment by bqx619** [comment]
目前的进度:

- 整理了 R1 阶段 80% 的 [需求和基础流程](https://www.notion.so/bqx619/Hacktuary-67b7a3ef74794ca5b79fd1be10dbbcd8)
- 根据需求, 拆分了 R1 阶段的需要做的事

欢迎对以上任意部分进行 _comment_ 或 _join_

**Comment by Cobb9527** [comment]
dorahacks算是竞品吗? 感觉做的也挺好的
